ftw/dashboard/dragndrop/browser/dashboard.py

```diff
@@ -1,9 +1,15 @@
-from Products.CMFCore.utils import getToolByName
 from plone.app.layout.dashboard.dashboard import DashboardView
 from plone.portlets.interfaces import IPortletType
+from Products.CMFCore.utils import getToolByName
 from zope.component import queryUtility
 
 
+try:
```

Comment:
Is there a better solution to check if the `createToken` method is available i.e. it's in version `2.0.2` or higher?